editor/src/viewport_tools/tool.rs

```diff
@@ -267,6 +267,7 @@ pub fn standard_tool_message(tool: ToolType, message_type: StandardToolMessageTy
 			ToolType::Path => Some(PathMessage::Abort.into()),
 			ToolType::Navigate => Some(NavigateMessage::Abort.into()),
 			ToolType::Pen => Some(PenMessage::Abort.into()),
+			ToolType::Freehand => Some(FreehandMessage::Abort.into()),
```

Comment:
Same request here as mentioned in the other comment: if you don't mind filling in (commented out) additional lines for all of the tools, then order them according to their order of appearance in the tool shelf (the order used in the block right above this block), that would be much appreciated!